index.js

```diff
@@ -41,6 +44,7 @@ module.exports = {
           .then(function(results) {
             var data = results.data;
             data.scm = results.scm;
+            self._tagVersion(results.data.revisionKey);
```

Comment:
Why not just use `data.revisionKey` for consistency? You used it below.

Reply:
You are right, let me change it....

BTW, do you think it would be nice to publish an addon `ember-cli-deploy-index-manipulation` that allows us to manipulate `index.html` file in any deploy hook? The current PR is almost impossible to get merged to upstream.... @archit opinion?

Reply:
Sorry, I don't have a good answer for that 😞